dataprep/eda/configs.py

```diff
@@ -0,0 +1,1042 @@
+"""
+Parameter configurations
+"""
+# pylint: disable=too-many-lines,no-self-use,blacklisted-name
+
+from __future__ import annotations
+
+from typing import Any, Dict, List, Optional, Tuple
+
+
+class Config:
+    """
+    Configuration class
+    """
+
+    # pylint: disable=too-many-instance-attributes
+
+    def __init__(self) -> None:
+        self.hist = Hist()
```

Comment:
For config classes I think maybe it's better to make it more declarative by using pydantic - we are already using it so there will be no additional dependencies.

The config class will become:
```python
from pydantic import BaseModel, Field
class Config(BaseModel):
    bar: Bar = Field(default_factory=Bar)
    pie: Pie = Field(default_factory=Pie)
    
    @classmethod
    def from_dict(...

```

Reply:
Same for `Bar`, `Pie` ...

Reply:
https://pydantic-docs.helpmanual.io/